Chaincase.UI/Components/Status.razor

```diff
@@ -12,6 +12,9 @@
     <IonItem>
         <IonLabel>Tor Status: @TorLabel()</IonLabel>
     </IonItem>
+    <IonItem>
+        <IonLabel>Tor Hidden Service Status: @TorHiddenServiceStatus()</IonLabel>
+    </IonItem>
```

Comment:
Because the server is fragile on iOS and requires the app to be open for success, perhaps a colorful full-screen warning that presents the QR is more appropriate. We must also encourage use of iOS [share extensions](https://developer.apple.com/design/human-interface-guidelines/ios/extensions/sharing-and-actions/) rather than the app switcher or else Tor gets killed.